### PR TITLE
docs(instance,Component,elements): always use component

### PR DESCRIPTION
**Pull request by vvo**
And put consistent usage of `<Component>` in docs instead of
`<Component/>`. fb/react does the same in their docs

### CODE REVIEW

File: docgen/src/Getting_started_React_native.md [hunk 1]
```diff
@@ -242,7 +242,7 @@ const SearchBox = connectSearchBox(({ refine, currentRefinement }) => {
 
 ```
 
-Let’s add this new `<SearchBox>` component to our InstantSearch instance. 
+Let’s add this new `<SearchBox>` component inside our `<InstantSearch>`.
```
Comment (Haroenv) [inline docgen/src/Getting_started_React_native.md]: maybe:

inside the/our `<InstantSearch>` component.

File: docgen/src/Getting_started_React_native.md [hunk 2]
```diff
@@ -316,14 +316,14 @@ const Highlight = connectHighlight(
 );
 ```
 
-Let’s use this new `<Highlight/>` component inside our `<Hits/>` template: 
+Let’s use this new `<Highlight>` component inside our `<Hits>` template:
```
Comment (Haroenv) [inline docgen/src/Getting_started_React_native.md]: here we use template, this should be "component" too

File: docgen/src/Getting_started_React_native.md [hunk 3]
```diff
@@ -427,7 +427,7 @@ const RefinementList = connectRefinementList(({ refine, items }) =>
 );
 ```
 
-Let’s add this new `<RefinementList>` component to our InstantSearch instance. 
+Let’s add this new `<RefinementList>` component to our `<InstantSearch>`.
```
Comment (Haroenv) [inline docgen/src/Getting_started_React_native.md]: same as earlier

File: docgen/src/Getting_started_React_native.md [hunk 4]
```diff
@@ -533,7 +533,7 @@ class Categories extends React.Component {
 }
 ```
 
-Second, let’s add our Modal to our first `<InstantSearch>` instance:
+Second, let’s add our Modal to our first `<InstantSearch>`:
```
Comment (Haroenv) [inline docgen/src/Getting_started_React_native.md]: component

File: docgen/src/guide/Virtual_widgets.md [hunk 5]
```diff
@@ -30,7 +30,7 @@ const App = () =>
 ```
 
 **Notes**:
-* The `<Hoodies/>` component is what we call a virtual widget.
+* The `<Hoodies>` component is what we call a virtual widget.
 * Virtual widgets allows you to pre refine any widget without rendering anything.
```
Comment (Haroenv) [inline docgen/src/guide/Virtual_widgets.md]: irrelevant, but 

```diff
- Virtual widgets allows you
+ Virtual widgets allow you
```

**Comment by vvo** [comment]
@Haroenv Next time you can do the edits directly, those are very good and valuable

**Comment by vvo** [comment]
Changes done, good to 🚢  thx @Haroenv

**Comment by Haroenv** [comment]
I wasn't 100% sure, wanted feedback on it @vvo

**Pull request merged**